README.md

```diff
@@ -90,11 +91,18 @@ controller.add(&hisThread);
 controller.add(&sensorReadings);
 ...
 ```
+or
+```c++
+// Instantiate a new StaticThreadController
+StaticThreadController<2> controller (Thread(my_callback, my_interval), Thread(his_callback, his_interval));
```

Comment:
New version won't be able to construct threads, it will accept only pre-created threads like ThreadController does.